index.js

```diff
@@ -0,0 +1,12 @@
+const API_URL = "https://api.github.com/users/";
```

Comment:
Could you please set the variable to `BASEURL`

Reply:
done